gtk-default-themes.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/gtk/gtksettingsprivate.h b/gtk/gtksettingsprivate.h
+index 47f75a6297..fd822c630a 100644
+--- a/gtk/gtksettingsprivate.h
++++ b/gtk/gtksettingsprivate.h
+@@ -23,8 +23,8 @@
+ 
+ G_BEGIN_DECLS
+ 
+-#define DEFAULT_THEME_NAME      "Adwaita"
+-#define DEFAULT_ICON_THEME      "Adwaita"
++#define DEFAULT_THEME_NAME      "io.elementary.stylesheet.blueberry"
++#define DEFAULT_ICON_THEME      "elementary"
```

Comment:
i don't think there's a non-intentional way to use the icon fallback on flatpak, the host icons folder is mounted on the sandbox, so the selected theme is always found.

Reply:
I mean, given the fact Dippi for instance automatically overrides even the icon theme, I wonder if Dippi's strategy can be incorporated directly into the platform.